controller.log

```diff
@@ -0,0 +1,9 @@
+2023-08-02 06:03:00 | INFO | controller | args: Namespace(host='localhost', port=21001, dispatch_method='shortest_queue')
```

Comment:
we can remove this file if it's not used by others.

Reply:
Sure！